tests/php/test-class-amp-theme-support.php

```diff
@@ -1445,6 +1445,18 @@ public function test_filter_admin_bar_style_loader_tag( $setup_callback, $assert
 		$assert_callback( new DOMXPath( $dom ) );
 	}
 
+	/**
+	 * Test filter_admin_bar_style_loader_tag when ->deps is not an array.
+	 *
+	 * @covers \AMP_Theme_Support::filter_admin_bar_style_loader_tag()
+	 */
+	public function test_filter_admin_bar_style_loader_tag_non_array() {
+		wp_enqueue_style( 'admin-bar' );
+		$GLOBALS['wp_styles']->registered['admin-bar']->deps = null;
+		$tag = 'foo';
```

Comment:
This should be an actual link tag as normally passed in. Otherwise the test is possibly failing silently.

Reply:
Ah, good point. Applied in https://github.com/ampproject/amp-wp/pull/4207/commits/68964a40ead0dced8d50465dc09cd71788363c6b